content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -332,15 +348,15 @@ The following configuration variables are available for both automatic and custo
 
 `DD_TRACE_AGENT_URL`
 : **TracerSettings property**: `Exporter.AgentUri`<br>
-Sets the URL endpoint where traces are sent. Overrides `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set. <br>
-**Default**: `http://<DD_AGENT_HOST>:<DD_TRACE_AGENT_PORT>`
+Sets the URL endpoint where traces are sent. Overrides `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.<br>
+**Default**: `http://<DD_AGENT_HOST>:<DD_TRACE_AGENT_PORT>` if they are set or `http://localhost:8126`
 
 `DD_AGENT_HOST`
-: Sets the host where traces are sent (the host running the Agent). Can be a hostname or an IP address. Ignored if `DD_TRACE_AGENT_URL` is set. <br>
+: Sets the host where traces are sent (the host running the Agent). Can be a hostname or an IP address. Prefer using `DD_TRACE_AGENT_URL` that has precedence over this parameter. <br>
```

Comment:
```suggestion
: Sets the host where traces are sent (the host running the Agent). Can be a hostname or an IP address. Prefers using `DD_TRACE_AGENT_URL` that has precedence over this parameter. <br>
```

Reply:
This is meant to mean "You should prefer using..." Maybe better phrasing will make this intent clearer?

Reply:
Great, thanks!